developer-tools.md

```diff
@@ -301,7 +301,7 @@ incorporated into a maintenance release. These should only be used by Spark deve
 may have bugs and have not undergone the same level of testing as releases. Spark nightly packages 
 are available at:
 
-- Latest master build: <a href="http://people.apache.org/~pwendell/spark-nightly/spark-master-bin/latest">http://people.apache.org/~pwendell/spark-nightly/spark-master-bin/latest</a>
```

Comment:
Yep. :) I did both ways in multiple times (Actually, the way to regenerate .html first.)
Since I observed that `jekyll serve` also regenerate this page automatically, I wrote like that as a single step.